doc/source/whatsnew/v1.5.0.rst

```diff
@@ -334,7 +334,7 @@ Other enhancements
 - :meth:`Series.add_suffix`, :meth:`DataFrame.add_suffix`, :meth:`Series.add_prefix` and :meth:`DataFrame.add_prefix` support a ``copy`` argument. If ``False``, the underlying data is not copied in the returned object (:issue:`47934`)
 - :meth:`DataFrame.set_index` now supports a ``copy`` keyword. If ``False``, the underlying data is not copied when a new :class:`DataFrame` is returned (:issue:`48043`)
 - The method :meth:`.ExtensionArray.factorize` accepts ``use_na_sentinel=False`` for determining how null values are to be treated (:issue:`46601`)
-
+- :meth:`Series.add_suffix`, :meth:`DataFrame.add_suffix`, :meth:`Series.add_prefix` and :meth:`DataFrame.add_prefix` support an ``axis`` argument. If ``axis`` is unset, rows are considered for ``Series`` and columns for ``DataFrame``. If ``axis`` is set, this default behaviour can be overwritten (:issue:`48085`)
```

Comment:
The new behaviour is consistent with the current behaviour?

Reply:
yup you're right. it's basically backward compatible.
- if user doesn't specify this ``axis`` parameter, it resorts to current behavior where rows are considered for ``Series`` and columns for ``DataFrame``.
- If user specifies this `axis` parameter, then the new behavior takes effect & add_prefix/add_suffix is done based on the axis provided.

Reply:
Then you can trim this a bit and just write that the default behavior can be overwritten without specifying what the current behavior is

Reply:
thanks @phofl for the suggestion. Makes sense.
will update this.

Reply:
Updated the whatsnew as per suggestion